programs-service/src/services/errors/entitySaveError.js

```diff
@@ -0,0 +1,15 @@
+const BaseError = require('@errors/baseError');
+
+class EntitySaveError extends BaseError {
```

Comment:
not saved? I think it's 500 or 400, let's keep to the HTTP status codes in errors.
Or we can make server side errors likewise db ones, and have HTTP status errors on the routing side, but I think it's an overkill.